src/opm/common/utility/numeric/calc_cellvol.cpp

```diff
@@ -0,0 +1,199 @@
+#include <opm/common/utility/numeric/calc_cellvol.hpp>
+
+
+/* 
+    Cell volume calculation based on following publication:
+   
+    D. K Pointing, Corner Point Geometry in Reservoir Simulation , 
+    ECMOR I - 1st European Conference on the Mathematics of Oil Recovery, 
+    1989 
+*/
+ 
+using namespace std;
```

Comment:
Even in a cpp file, `using namespace std` should in general be avoided. Usually in OPM we use the `std::` prefix whenever needed. If that would create major readability problems, the more limited `using std::some_function_you_need_to_call_a_lot` is ok.